docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -135,6 +153,38 @@ async function getLocationAsync() {
 }
 ```
 
+## Permissions response
+
+An object with information about the permissions, including status, expiration, and scope (if applicable).
+The top-level `status`, `expires` and `canAskAgain` keys are a reduction of the values from each individual permission.
+If all single permissions have a `status` of `'denied'`, then that the top level `status` is `denied`; in other words, the top-level `status` is `'granted'` if and only if all of the individual permissions are `'granted'`. Otherwise, `status` is`undetermined`.
```

Comment:
Yeah, it was kind of "a lot". That's why I moved it to its own section and referred to that as much as possible. I'll wait for this one for cruzan to double-check. I think it makes sense what you wrote 😄

Reply:
I would rearrange the order of the Permissions response block to:
```
An object with information about the permissions, including status, expiration, and scope (if applicable). 

<code block showing response object>

explanation of the top level keys
```

With the explanation of the top level keys being:
```
The top-level `status`, `expires`, and `canAskAgain` keys depend on the values returned for each of the individual permission requests:
- If **all single permissions** have a `status` of 'denied', then the top level `status` is `denied`. If **all single permissions** have a `status` of 'granted', then the top level `status` is `granted`. Otherwise (if some single permissions return `granted` and others return `denied`) the top level `status` is `undetermined`. 
- If every single permission has a `status` of `granted`, then the top-level `granted` field is `true`. Otherwise, it is `false`. 
- The top-level `expires` field matches the value of the earliest expiring permission.
```

Reply:
I made a quick test app for this, turns out `undetermined` is prioritized before `denied`. So I adjusted your suggestion, hope this make things clear again @cruzach 😄